### ISSUE TITLE
Some arrow characters are not showing up

### ISSUE THREAD

**Issue by naps62** [michaeldfallen/git-prompt#50]
Hi, Linux user here.

Some of the characters are now showing up correctly. Particularly the "Remote commits status" arrow icons:

![screenshot from 2015-09-01 12-03-12](https://cloud.githubusercontent.com/assets/283819/9602538/7387298a-50a1-11e5-95ea-d66c6e985f48.png)

My font is currently "Adobe Source Pro for Powerline", although I already tried a couple others, including non-Powerline fonts.

I'm not sure if this is some issue with git-radar, or with my own system, but maybe someone here has had this problem before?

**Comment by michaeldfallen** [comment]
Does your terminal support utf-8? If you are using Powerline fonts it probably does. I've tested it on my terminal (Mac OSX 10.9.5 apple Terminal) using the Sourcecode Pro Powerline provided by https://github.com/powerline/fonts and it seems to work fine. Are you using a special build of Source code?

**Comment by naps62** [comment]
@michaeldfallen I'm not using any special build, although I don't remember exactly how it is that I installed it

But as I said, I tried to use the default `Adobe Source Pro`, and other fonts such as `Source Code Pro` and even `Arial`.
The issue occurs with all of them

**Comment by naps62** [comment]
Update: I just saw this now for the first time:

![screenshot from 2015-09-01 17-38-11](https://cloud.githubusercontent.com/assets/283819/9610271/4f59bdb4-50d0-11e5-8a88-3c377441bb35.png)

So that arrow icon is working after all. I'm messing the missing icon there is the italic `m` that shows up in the readme

**Comment by michaeldfallen** [comment]
Which terminal are you using? Does it support xterm 256 colours (I'm guessing yes since you have colour in your screenshot) and utf-8 as default character encoding?

**Comment by michaeldfallen** [comment]
My settings (and default for OSX I think) are:
<img width="388" alt="screen shot 2015-09-02 at 10 46 11" src="https://cloud.githubusercontent.com/assets/1446145/9627976/c0c0637e-515f-11e5-9ea9-96ad080c1552.png">

**Comment by naps62** [comment]
I'm using uxterm which is suposed to be xterm with support for utf-8.

However, I just tried installing terminator, and everything seems to work well there. So the issue only happens on xterm/uxterm, and only with two characters it seems (the italic 'm', and the double arrow)

I was researching a bit about this problem, but there doesn't seem to be a lot of good info on uxterm about this. I also just noticed I can't get japanese characters to work, for example.

Anyway, this seems to be an issue with uxterm (or with my uxterm configs), so I'll close this
